zerver/lib/test_classes.py

```diff
@@ -652,8 +652,12 @@ def send_and_test_private_message(self, fixture_name: Text, expected_subject: Te
         if content_type is not None:
             kwargs['content_type'] = content_type
 
-        msg = self.send_json_payload(self.test_user, self.url, payload,
-                                     stream_name=None, **kwargs)
+        if kwargs.get('sender') is not None:
+            msg = self.send_json_payload(kwargs['sender'], self.url, payload,
+                                         stream_name=None, **kwargs)
+        else:
+            msg = self.send_json_payload(self.test_user, self.url, payload,
+                                         stream_name=None, **kwargs)
```

Comment:
I would prefer to write this as `sender = kwargs.get('sender', self.test_user)`.